src/main/java/com/powsybl/openloadflow/network/LfSecondaryVoltageControl.java

```diff
@@ -41,7 +42,45 @@ public LfBus getPilotBus() {
         return pilotBus;
     }
 
-    public Set<LfBus> getControlledBuses() {
-        return controlledBuses;
+    public Set<GeneratorVoltageControl> getGeneratorVoltageControls() {
+        return generatorVoltageControls;
+    }
+
+    private static List<LfBus> findControllerBuses(LfBus controlledBus) {
```

Comment:
Do we have forgotten `MAIN` status filtering?

Reply:
I think the check is done when listing the controlled buses